notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.                sample = torch.tensor(result_numpy, dtype=self.dtype)
</pre><p>torch.from_numpy(result_numpy)</p><p> </p><p>instead of class, can we use namedtuple? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>This is clever! done</p>